core/src/main/scala/pureconfig/package.scala

```diff
@@ -78,7 +46,7 @@ package object pureconfig {
    *         isn't possible
    */
   def loadConfig[Config](path: Path)(implicit reader: Derivation[ConfigReader[Config]]): ConfigReader.Result[Config] =
-    loadConfig(path, "")
+    ConfigSource.file(path).load[Config]
```

Comment:
This changes the behavior, since it doesn't load `reference.conf`. The same goes for a lot of the functions in this file.

Reply:
Well spotted. I'll create a test case to make sure that we don't miss this again and then I'll fix it.